src/pybind/mgr/nfs/utils.py

```diff
@@ -101,4 +101,4 @@ def _get_cephfs_client() -> CephfsClient:
         stx = fs_handle.statx(path.encode('utf-8'), cephfs.CEPH_STATX_MODE,
                               cephfs.AT_SYMLINK_NOFOLLOW)
         if not stat.S_ISDIR(stx.get('mode')):
-            raise NotADirectoryError()
```

Comment:
Are you sure about his? I understand this is still draft, but...

Reply:
https://github.com/ceph/ceph/pull/56759#issuecomment-2058566736